assets/js/components/Dashboard/PaMessaging/PaMessagesPage.tsx

```diff
@@ -27,10 +28,12 @@ const PaMessagesPage: ComponentType = () => {
           <Col className="pa-message-table-container">
             <Row className="pa-message-table-action-bar">
               <Col>
-                <Button>
-                  <PlusCircleFill className="pa-message-table-action-bar__plus" />{" "}
-                  Add New
-                </Button>
+                <Link to="/pa-messages/new">
+                  <Button>
```

Comment:
Nesting links and buttons isn't proper markup. If we want a link that looks like a button, it should be possible to apply the class names to the link to give it that appearance.